odd-platform-api/src/main/java/com/provectus/oddplatform/dto/TagDto.java

```diff
@@ -0,0 +1,17 @@
+package com.provectus.oddplatform.dto;
+
+import java.time.LocalDateTime;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+@Value
+@RequiredArgsConstructor
+public class TagDto {
```

Comment:
Why do you need the dto? Jooq's pojo seems to have all of the fields present already

Reply:
Jooq's dto is missing the 'usedCount' field which is not a native one.